kubernetes/course-expired-cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: course-expired
+  schedule: "0 0 1 * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      restartPolicy: OnFailure
+          containers:
+            - name: create-user-quiz-states
+              image: ${BACKEND_IMAGE}
+              command: ["node", "run", "update-expired-courses"]
```

Comment:
```
npm run update-expired-courses
```
package.json: https://github.com/rage/quizzes/blob/master/packages/backendv2/package.json#L17

change all names to update-expired-courses